saleor/graphql/notifications/inputs.py

```diff
@@ -0,0 +1,11 @@
+import graphene
+
+
+class ExternalNotificationTriggerInput(graphene.InputObjectType):
+    ids = graphene.List(graphene.ID, required=True, description="List of customer ids.")
+    extra_payloads = graphene.JSONString(
+        required=True, description="Additionaly payload."
+    )
+    external_event_type = graphene.String(
+        required=True, description="External event type description."
```

Comment:
Can we describe this arg properly? What values does it expect? From what I see it can accept Sendgrid template IDs, what other values does it accept? We should explain it.